extensions/wikia/AdEngine/js/video/videoSettings.js

```diff
@@ -20,6 +21,7 @@ define('ext.wikia.adEngine.video.videoSettings', [
 			state.autoPlay = isAutoPlay(params);
 			state.splitLayout = Boolean(params.splitLayoutVideoPosition);
 			state.moatTracking = Boolean(params.moatTracking);
+			state.withUiControls = Boolean(params.hasUiControls);
```

Comment:
Doing that only once has very low impact on overall performance, it's not done in loop. For me static constructor is better, because this not stops from further optimisation done by browser JS engines that might be done. `!!` on the other hand does not have such guarantee.